conf/prod.yml

```diff
@@ -0,0 +1,4 @@
+$include:
+  - conf/base.yml
+
+DAC.quantizer_dropout: 0.5
```

Comment:
This change is already available as `conf/final/44khz.yml`. Perhaps that config can be made better because i'd just copy pasted the base.yml and made mods